desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -690,6 +690,18 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
                 </label>
               </div>
 
+              <div class="control-group">
+                <label class="checkbox inline-block">
+                  <input type="checkbox" data-bind="checked: isOzone"> ${_('IsOzone')}
+                </label>
+              </div>
+
+              <div class="control-group" data-bind="visible: isOzone">
+                <label class="control-label"><div>${ _('Ozone Bucket Path') }</div>
+                  <input type="text" class="form-control input-xxlarge" data-bind="value: ozonefsPath,  enable: isOzone, valueUpdate: 'afterkeydown'" placeholder="${ _('ofs://ozone1/vol/bucket') }" required>
```

Comment:
This might be a stupid question but I'm trying to understand this UI. The user selects any file from HDFS or elsewhere in step 1 (nothing about ozone there) and then in step 2 after expanding the "extras" section is presented with the option to check `isOzone` and then has to provide the ozonefsPath... When I look at this flow this seems backward. The output is a table but we providing a path to an ozone bucket?